packages/docsearch-react/src/DocSearchModal.tsx

```diff
@@ -239,6 +241,7 @@ export function DocSearchModal({
 
               return Object.values<DocSearchHit[]>(sources).map((items) => {
                 return {
+                  sourceId: 'DocSearchHits',
```

Comment:
All the DocSearch section will get the same `sourceId`s here. Can we differentiate them?

Reply:
Fixed in https://github.com/algolia/docsearch/pull/999/commits/fc965daeb4877a3febd619012dff39febc7f5de7